platforms/ios/HelloCordova.xcarchive/Products/Applications/HelloCordova.app/Frameworks/Sonect.framework/Headers/SNCConfiguration.h

```diff
@@ -0,0 +1,65 @@
+//
+//  SNCConfiguration.h
+//  SonectSDK
+//
+//  Created by Bruno Sousa on 21/12/2018.
+//  Copyright © 2018 Sonect. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SNCConfiguration : NSObject
+
+/**
+ A 2 digit ISO country code i.e. "ch" for Switzerland
+ */
+@property (nonatomic, nullable, copy, readonly) NSString* alpha2CountryCode;
+
+/**
+ The user's preferred currency i.e. "CHF" for Swiss Francs
+ */
+@property (nonatomic, nullable, copy, readonly) NSString* currency;
+
+/**
+ Set allowed country codes.
+ Example: would be US and Switzerland
+ @[@1, @41]
+ */
+@property (nonatomic, nullable, copy, readonly) NSArray <NSNumber *> *allowedCountryCodes;
```

Comment:
Maybe better to use enum in configuration to let us control available countries

Reply:
We have a configuration `plist` file that feeds these values, and the configuration file should eventually be created in textual form (plist) by our partner platform, similar to the way Google works... We could create an enum from raw values, but I'm skeptical about the value of making an enum with 195 values for country codes in each of our code bases. I'll let the server deal with erroneous input ;)